pkgs/top-level/python-packages.nix

```diff
@@ -4414,6 +4414,8 @@ self: super: with self; {
 
   ical = callPackage ../development/python-modules/ical { };
 
+  ical2orgpy = callPackage ../development/python-modules/ical2orgpy { };
```

Comment:
Reading upstream's README, this seems more like a script rather then a python module, hence it should be in `all-packages.nix`.

When moving it to there, use `callPackage` (and not `python3Packages.callPackage` and use `python3Packages.buildPythonPackage` in there.